docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -51,6 +51,15 @@ function template_preprocess_uiowa_core_headline(&$variables) {
 
 }
 
+/**
+ * Implements hook_library_info_alter().
+ */
+function uiowa_core_library_info_alter(array &$libraries, string $extension): void {
+  if ($extension == 'claro') {
```

Comment:
I don't get this. This supposed to be claro?

Reply:
The base theme for Gin is Claro (https://git.drupalcode.org/project/gin/-/blob/8.x-3.x/gin.info.yml#L3.), which is why I was targeting claro to override some weird styles that are happening.